lnrpc/marshall_utils.go

```diff
@@ -179,3 +180,57 @@ func MarshalOutPoint(op *wire.OutPoint) *OutPoint {
 		OutputIndex: op.Index,
 	}
 }
+
+// Parse the coin selection strategy string from the command line
+// to an equivalent lnrpc.CoinSelectionStrategy enum type.
+func ParseCoinSelectionStrategyString(strategy string)(
+	*CoinSelectionStrategy, error) {
+
+	var coinSelectionStrategy CoinSelectionStrategy
+	switch strategy {
+	case "global-config":
+		coinSelectionStrategy = CoinSelectionStrategy_STRATEGY_USE_GLOBAL_CONFIG
+
+	case "largest":
+		coinSelectionStrategy = CoinSelectionStrategy_STRATEGY_LARGEST
+
+	case "random":
+		coinSelectionStrategy = CoinSelectionStrategy_STRATEGY_RANDOM
+
+	default:
+		return nil, fmt.Errorf("unknown coin selection strategy "+
+			"%v", strategy)
+	}
+
+	return &coinSelectionStrategy, nil
+}
+
+// Parse the coins selection strategy lnrpc.CoinSelectionStrategy enum type
+// to an equivalent wallet.CoinSelectionStrategy type.
+func ParseCoinSelectionStrategyEnum(strategy CoinSelectionStrategy)(
```

Comment:
This kind of parsing is what we usually call "Unmarshaling" in the context of converting between RPC and native types. So this can be somewhat shorter too, just `UnmarshalCoinSelectionStrategy`.